package.json

```diff
@@ -5,7 +5,7 @@
     "build": "parcel build src/index.html",
     "lint": "npm run lint:js && npm run lint:css",
     "lint:js": "eslint ./src/js",
-    "lint:css": "stylelint './src/css/**/*.scss'",
+    "lint:css": "stylelint \"./src/css/**/*.scss\"",
```

Comment:
Fix a compatibility issue in Windows: https://github.com/stylelint/stylelint/issues/3624#issuecomment-416876570